party/models.py

```diff
@@ -102,4 +104,3 @@ def retrieve_all_party_names_and_ids(self):
             handle_record_not_found_exception(e, logger=logger)
 
         return party_items_list
```

Comment:
Hi @tejasaditya I would like to maintain the standard of a new line at the end of the file. I'm curious if your IDE gave you a warning to remove this linebreak?

Reply:
Hi @DaleMcGrew. It looks like my IDE does have line 107 as a blank line but it doesn't match what I see in the repository online. PyCharm did warn me about having a line so I added it. I also have no unstaged/uncommitted changes

Reply:
My apologies, I read the .github diff wrong.